frontend/src/components/SearchTool/SearchTool.js

```diff
@@ -0,0 +1,110 @@
+import { FaSearch, FaCheck } from 'react-icons/fa';
+import Select, { components } from 'react-select';
+import { useQuery } from 'react-query';
+import {
+  BoardName,
+  CheckIcon,
+  LeftSideOfOptionRow,
+  OptionRow,
+  RightSideOfOptionRow,
+  StyledOptions,
+  Title,
+  Container,
+} from './SearchTool.styled';
+import { UserAvatar } from 'components/UserAvatar/UserAvatar';
+
+export const SearchTool = (callback, userID) => {
+  const {
+    data: dataAvailableForUserBoards,
+    isLoadingAvailableBoards,
+    isErrorAvailableBoards,
```

Comment:
Możesz się upewnić, czy to działa u Ciebie prawidłowo. 
Ja jak zmieniłem nazwy na swoje to nie działało, więc być może jest potrzebne coś w stylu: 
_isLoading: isLoadingAvailableBoards,
isError: isErrorAvailableBoards,_